### ISSUE TITLE
Hi！How to use my own data set for training? 

### ISSUE THREAD

**Issue by LiBiGo** [NVlabs/DiscoBox#9]
How to use my own data set for training? I didn't see relevant information in the description.

**Comment by LiBiGo** [comment]
2022-08-28 16:59:26,968 - mmdet - INFO - Saving checkpoint at 2 epochs
[>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>] 84/84, 3.3 task/s, elapsed: 25s, ETA:     0s
2022-08-28 16:59:56,177 - mmdet - INFO - Evaluating bbox...
Loading and preparing results...
2022-08-28 16:59:56,178 - mmdet - ERROR - The testing results of the whole dataset is empty.
ERROR:mmdet:The testing results of the whole dataset is empty.
2022-08-28 16:59:56,180 - mmdet - INFO - Exp name: discobox_solov2_r101_dcn_fpn_3x.py
2022-08-28 16:59:56,180 - mmdet - INFO - Epoch(val) [2][96]

#####################################################################################
**2022-08-28 16:59:56,178 - mmdet - ERROR - The testing results of the whole dataset is empty.
ERROR:mmdet:The testing results of the whole dataset is empty.** 

How to solve this error? Is there a problem with my dataset settings?
###############################################################################

**Comment by voidrank** [comment]
Hi @DaDogs ,

It seems that the validation set is empty.

Best,

Shiyi

**Comment by MIYU8305** [comment]
Hi, DaDogs and voidrank

When I trained own dataset(only have bounding box annoation), 
I got the log result as below
2022-09-15 08:17:04,343 - mmcv - INFO - Reducer buckets have been rebuilt in this iteration.
2022-09-15 08:17:40,474 - mmdet - INFO - Saving checkpoint at 2 epoch

I would like to ask that can I train model without segmentation annotation using this source-code?
